src/test/java/uk/gov/homeoffice/digital/sas/timecard/utils/TimeEntryFactory.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.homeoffice.digital.sas.timecard.utils;
+
+import java.util.Date;
+import java.util.UUID;
+import uk.gov.homeoffice.digital.sas.timecard.model.TimeEntry;
+
+public class TimeEntryFactory {
```

Comment:
could we make this class final please, keeps it as a static util based class.

Reply:
Updated